unitary/examples/tictactoe/tic_tac_toe.py

```diff
@@ -244,3 +244,71 @@ def print(self) -> str:
             if idx in [2, 5, 8] and row != 2:
                 output += "--------------------------\n"
         return output
+
+class Game:
+    def __init__(self, game):
+        self.game = game
+
+    def setup(self):
+        print("Welcome to quantum tic tac toe!")
+        print("Here is the board:")
+        self.printBoardMap()
+        self.player = "X"
+        self.playerQuit = False
+
+    def handleMove(self, move): 
+        if move == 'map':
```

Comment:
Should we change move to an enum so that the list of commands is clear?

Reply:
Done.